packages/shared/package.json

```diff
@@ -14,7 +14,10 @@
     "clean": "rimraf dist coverage",
     "test": "jest"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@mikro-orm/core": "^5.7.14",
+    "@hangar/database": "*"
```

Comment:
Ooooh we can get fancy with this... can you actually move this to dev dependencies? I'll make another related comment below 👀